docs/guides/how-to-create.md

```diff
@@ -1,52 +1,59 @@
-  - the [Send SMS capability](https://superface.ai/communication/send-sms@1.0.1) contains two use-cases: _Send SMS message_ and _Retrieve Message Status_.
-  - for the [Geocoding capability](https://superface.ai/address/geocoding@3.0.1) contains two use-cases: _Geocode address_ and _Reverse geocode_.
-  - for the [Shipment information](https://superface.ai/delivery-tracking/shipment-info@1.0.1) contains a single use-case: _Retrieve Shipment Status_.
-- **Map** is a file with `.suma` extension written in the Comlink language. It connects the _Use-Case_ to a _Provider_. It contains the provider-specific logic to fulfill the use-case: what API endpoints need to be called, how to format requests and responses, and how an authentication is performed.
-  - For example the [Send Email](https://superface.ai/communication/send-email) capability can be fulfilled by many providers, e.g. Mailchimp, SendGrid, Postmark, and others.
+- **Use Case** represents a functionality that serves a business need. For example: send email, geocode a postal address, or track a package. Use case describes its expected input and result data. Use cases are listed in _Profile_ files with `.supr` extension. For example:
+  - the [Send SMS profile](https://superface.ai/communication/send-sms@1.0.1) profile contains two use cases: _Send SMS message_ and _Retrieve Message Status_.
+  - the [Geocoding profile](https://superface.ai/address/geocoding@3.0.1) profile contains two use cases: _Geocode address_ and _Reverse geocode_.
+  - the [Shipment information profile](https://superface.ai/delivery-tracking/shipment-info@1.0.1) profile contains a single use case: _Retrieve Shipment Status_.
+- **Profile** is collection of _use cases_ that serves common cause. Profiles are files with extension `.supr`.
+- **Map** connects the _Use Case_ to a _Provider_. It contains the provider-specific logic to fulfill the use case: what API endpoints need to be called, how to format requests and responses, and how an authentication is performed. Map is a file with `.suma` extension.
+  - For example the [Send Email](https://superface.ai/communication/send-email) use cases can be fulfilled by many providers, e.g. Mailchimp, SendGrid, Postmark, and others.
   - For each of these providers there is a separate `.suma` file which maps the profile to the respective provider.
-- **Provider** is a JSON file describing a set of host URLs and security schemes that fulfill the capability. One provider can be shared among multiple capabilities.
+- **Provider** is a JSON file describing a set of host URLs, security schemes and integration parameters that fulfill use cases. One provider can be shared among multiple use cases and profiles.
 
-<!-- TODO: Fancy diagram here -->
+![Comlink key concepts.](../../assets/comlink-key-concepts.png)
 
 ## What is Your Goal?
```

Comment:
I originally wrote the titles with US capitalization in mind, but this isn't the preferred style. Please, fix the headings to have our typical capitalization with only the first letter.

Reply:
fixed